<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_enviar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulsa boton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>contador</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>autoincremental</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>incremental proceso independiente</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoy_un_Men">
    <property name="title">
     <string>Soy un Menú</string>
    </property>
    <widget class="QMenu" name="menuSub_Men">
     <property name="title">
      <string>Sub Menú</string>
     </property>
     <addaction name="actionOtra_opci_n"/>
    </widget>
    <addaction name="menuSub_Men"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSoy_un_Men"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOtra_opci_n">
   <property name="text">
    <string>Otra opción</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
